src/types/transactions/index.ts

```diff
@@ -16,14 +16,26 @@ import {
   ApplicationClearStateTransaction as AppClearStateTxn,
   ApplicationNoOpTransaction as AppNoOpTxn,
 } from './application';
-
+import SuggestedParamsRequest from '../../client/v2/algod/suggestedParams';
```

Comment:
Why is this export needed?

Reply:
![testfg](https://user-images.githubusercontent.com/26249903/124717076-674e5100-df22-11eb-95b7-6555d70baa3e.png)

this type is being used while testing, a stub is used to override `getTransactionParams` function

Reply:
I would recommend using `ReturnType<algosdk.Algodv2['getTransactionParams']>` instead of exporting `SuggestedParamsRequest` if this is the only reason, since if we export `SuggestedParamsRequest`, we should probably export all of the internal request builder types, and that is a larger task that I'm not sure the library should do at this point.

Reply:
okay, thanks. this works

Reply:
removed.